presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -962,6 +962,8 @@ public void testAggregationFilter()
                         "FROM (SELECT count(*) FILTER (WHERE true) AS b)",
                 "SELECT 1");
 
+        assertQuery("SELECT count(1) FILTER (WHERE orderstatus = 'O') FROM orders", "SELECT 7333");
```

Comment:
Make the expected query be `SELECT count(*) FROM orders WHERE orderstatus = 'O'` instead of a hard coded value